pages/k8s/quickstart.md

```diff
@@ -30,156 +31,133 @@ With this quick start guide and some tools from Canonical, you'll have a Kuberne
   - [Rackspace][cloud-rackspace]
 
 <div class="p-notification--positive"><p markdown="1" class="p-notification__response">
-<span class="p-notification__status">Note:</span> If you don't meet these requirements, there are additional ways of installing the <emphasis>Charmed Distribution of Kubernetes<sup>&reg;</sup></emphasis>, inluding additional OS support and an entirely local deploy. Please see the more general <a href="/kubernetes/install">Installing CDK</a> page for details. </p></div>
+<span class="p-notification__status">Note:</span> If you don't meet these requirements, there are additional ways of installing the <emphasis>Charmed Distribution of Kubernetes<sup>&reg;</sup></emphasis>, inluding additional OS support and an entirely local deploy. Please see the more general <a href="/kubernetes/docs/install-manual">Installing Charmed Kubernetes</a> page for details. </p></div>
+
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-12">
+      <ol class="p-stepped-list--detailed">
+
+        <li class="p-list-step__item col-12">
+          <h3 class="p-list-step__title col-12"><span class="p-list-step__bullet">1</span>Install Juju</h3>
+          <div class="p-list-step__content">
+
+<a class="p-link--external" href="https://jaas.ai" > Juju</a> is a tool for
```

Comment:
Seems odd to have whitespace before Juju here.

Reply:
yeah, that's a typo. Think yourself lucky you don't have to write charms in HTML

Reply:
Speaking of, we have inluding up here too.